template/page-events.html.jinja

```diff
@@ -499,19 +498,38 @@
     "image": "AMAs.png",
     "end_date": "October 17, 2023"
     },
-]
-%}
-    {# 
-    Needs Links
     { "name": "XRPL Accelerator Demo Day",      
     "description": "Join us for XRPL Accelerator Demo Day in Singapore! Explore pitches from 11 promising startups building on the XRP Ledger, network with founders and investors, and kickstart the Singapore FinTech Festival.",
     "type": "meetup",
-    "link": "",
-    "location": "Hybrid (In-person: Singapore Central Business District. Virtual attendance: Webinar link)",
+    "link": "https://www.eventbrite.co.uk/e/xrpl-demo-day-tickets-740650023157?aff=oddtdtcreator",
+    "location": "Hybrid Singapore/Virtual Webinar",
```

Comment:
The Eventbrite link shows this as an in-person event only. Are we expecting the webinar link to be added later?